sites/kit.svelte.dev/src/routes/+layout.svelte

```diff
@@ -42,6 +78,46 @@
 			<span class="small">GitHub</span>
 			<span class="large"><Icon name="github" /></span>
 		</NavItem>
+
+		<li aria-hidden="true"><span class="separator" /></li>
+
+		<NavItem title="Dark Mode toggle">
+			<span on:click={toggleTheme}>
+			{#if currentTheme === 'theme-default'}
+				<span class="small">Dark mode</span>
+				<span class="large">
+					<svg
+						width="20"
+						height="20"
+						viewBox="0 0 24 24"
+						class="icon"
+					>
+							<path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z" />
+					</svg>
+				</span>
+			{:else}
+				<span class="small">Light mode</span>
+				<span class="large">
+					<svg
+						width="20"
+						height="20"
+						viewBox="0 0 24 24"
+						class="icon"
+					>
+						<circle cx="12" cy="12" r="5" />
+						<line x1="12" y1="1" x2="12" y2="3" />
+						<line x1="12" y1="21" x2="12" y2="23" />
+						<line x1="4.22" y1="4.22" x2="5.64" y2="5.64" />
+						<line x1="18.36" y1="18.36" x2="19.78" y2="19.78" />
+						<line x1="1" y1="12" x2="3" y2="12" />
+						<line x1="21" y1="12" x2="23" y2="12" />
+						<line x1="4.22" y1="19.78" x2="5.64" y2="18.36" />
+						<line x1="18.36" y1="5.64" x2="19.78" y2="4.22" />
```

Comment:
These icons have been taken from [feathericons](https://feathericons.com/), as it was the library used for all the other icons, but usage was limited, since the site-kit project has been archived and not maintained. Please share if there's a cleaner way to add the "sun" & "moon" icons